SharkCage/SharkCage.sln

```diff
@@ -33,8 +37,8 @@ Global
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
 		{569449B5-18D8-43F0-A6A0-7E754EFEBC25}.Debug|Any CPU.ActiveCfg = Debug|Win32
 		{569449B5-18D8-43F0-A6A0-7E754EFEBC25}.Debug|Any CPU.Build.0 = Debug|Win32
-		{569449B5-18D8-43F0-A6A0-7E754EFEBC25}.Debug|x64.ActiveCfg = Debug|Win32
-		{569449B5-18D8-43F0-A6A0-7E754EFEBC25}.Debug|x64.Build.0 = Debug|Win32
+		{569449B5-18D8-43F0-A6A0-7E754EFEBC25}.Debug|x64.ActiveCfg = Release|Win32
+		{569449B5-18D8-43F0-A6A0-7E754EFEBC25}.Debug|x64.Build.0 = Release|Win32
```

Comment:
Debug + x64 configuration now builds the Release version of the service again...
![image](https://user-images.githubusercontent.com/16246147/42123374-5d429590-7c51-11e8-8844-5bd266e0087f.png)